### DISCUSSION TITLE
Cloudflare Tunnel: ⚠ Something went wrong

### DISCUSSION THREAD

**Discussion by thiagvncs** [Radiokot/umbrel-cloudflared#13] [General]
I did everything correctly with the configuration and when I access my domain, a black background appears with the following phrase: ⚠️ Something went wrong
![umbrel os erro](https://github.com/user-attachments/assets/f500aeff-4a32-408f-b352-a2b1bdf59fd1)

**Comment by Radiokot** [comment]
The Umbrel interface seems incompatible with access through a tunnel
